packages/field/src/DisabledMixin.js

```diff
@@ -0,0 +1,118 @@
+import { dedupeMixin } from '@lion/core';
+
+/**
+ * #DisabledMixin
+ *
+ * For now this is a private mixin for option
+ * Do not use it anywhere else - it will become part of field eventually.
+ *
+ * @polymerMixin
+ * @mixinFunction
+ */
+export const DisabledMixin = dedupeMixin(
+  superclass =>
+    // eslint-disable-next-line no-shadow
+    class DisabledMixin extends superclass {
+      static get properties() {
+        return {
+          disabled: {
+            type: Boolean,
+            reflect: true,
+          },
+          // we us a property here as if we use the native tabIndex we can not set a default value
+          // in the constructor as it synchronously sets the attribute which is not allowed in the
+          // constructor phase
+          tabIndex: {
+            type: Number,
+            reflect: true,
+            attribute: 'tabindex',
+          },
+        };
+      }
+
+      static get disableMixinManagesTabIndex() {
+        return true;
+      }
```

Comment:
do I understand correctly that it is difficult to split the logic of disabled and tabindex, so you decided to have them in one? can we then maybe consider a different name for this mixin?

I see `disableMixinManagesTabIndex` flag, but it is enabled by default which should be reflected in the name

or otherwise maybe make it `false` by default and keep the name `DisabledMixin`?

Reply:
turned out that I misunderstood the name of the method, so we also agreed to rename this to `managesTabIndexViaDisableMixin` where `ViaDisableMixin` - is a suffix to namespace the method, because `managesTabIndex` is too generic

and indeed the value will be `false` by default, and then we can keep the name `DisabledMixin`

Reply:
changed 💪